hbase-server/src/main/java/org/apache/hadoop/hbase/tool/CanaryTool.java

```diff
@@ -128,14 +124,8 @@
 public class CanaryTool implements Tool, Canary {
   public static final String HBASE_CANARY_INFO_PORT = "hbase.canary.info.port";
 
-  public static final int DEFAULT_CANARY_INFOPORT = 16050;
-
-  public static final String HBASE_CANARY_INFO_BINDADDRESS = "hbase.canary.info.bindAddress";
```

Comment:
Why removing the static final declaration and use the string literal directly?

Reply:
Both defines are used in one place only. These defines seem superfluous when one-use only.  What you thinking?

Reply:
The DEFAULT_CANARY_INFOPORT is not used anywhere anymore after this change.

Let me put back HBASE_CANARY_INFO_BINDADDRESS so you'll +1 on this and I can get it in. Thanks.

Reply:
I think it's fine to remove these constants, at least to make them private. Anyway, I see no reason for them to be public.